packages/in-app-notification-dispatcher/src/templates/inAppTemplates.ts

```diff
@@ -15,101 +80,120 @@ export const inAppTemplates = {
+      version ?? ""
+    } dell'e-service <strong>${eserviceName}</strong>, a cui sei iscritto.`,
   eserviceDescriptorDocumentDeletedToConsumer: (
     eserviceName: string,
-    documentName = ""
-  ): string =>
-    `Il documento <strong>${documentName}</strong> dell'e-service <strong>${eserviceName}</strong> è stato rimosso.`,
-  agreementSuspendedUnsuspended: (
-    action: "sospeso" | "riattivato" | "archiviato",
-    subjectName: string,
-    eserviceName: string
-  ): string =>
-    `${subjectName} ha ${action} la richiesta di fruizione relativa all'e-service <strong>${eserviceName}</strong>.`,
-  agreementManagementToProducer: (
-    consumerName: string,
-    eserviceName: string,
-    action: "attivato" | "creato" | "aggiornato"
-  ): string =>
-    `${consumerName} ha ${action} la richiesta di fruizione relativa all'e-service <strong>${eserviceName}</strong>.`,
-  agreementActivatedRejectedToConsumer: (
-    producerName: string,
-    eserviceName: string,
-    action: "attivato" | "rifiutato"
+    documentName = "",
+    producerName: string
   ): string =>
-    `${producerName} ha ${action} la richiesta di fruizione relativa all'e-service <strong>${eserviceName}</strong>.`,
+    `L'ente erogatore <strong>${producerName}</strong> ha rimosso il documento <strong>${documentName}</strong> dell'e-service <strong>${eserviceName}</strong>.`,
   delegationApprovedRejectedToDelegator: (
+    eserviceName: string,
     delegateName: string,
     eventType: DelegationApprovedRejectedToDelegatorEventType,
     rejectionReason: string | undefined
   ): string => {
     const { action, delegationKind, reason } = match(eventType)
       .with("ProducerDelegationApproved", () => ({
         action: "approvato",
-        delegationKind: "erogazione",
-        reason: "",
+        delegationKind: "all'erogazione",
+        reason: "La delega è ora attiva.",
       }))
       .with("ConsumerDelegationApproved", () => ({
         action: "approvato",
-        delegationKind: "fruizione",
-        reason: "",
+        delegationKind: "alla fruizione",
+        reason: "La delega è ora attiva.",
       }))
       .with("ProducerDelegationRejected", () => ({
         action: "rifiutato",
-        delegationKind: "erogazione",
+        delegationKind: "all'erogazione",
         reason: rejectionReason ? ` Motivo: ${rejectionReason}.` : "",
       }))
       .with("ConsumerDelegationRejected", () => ({
         action: "rifiutato",
-        delegationKind: "fruizione",
+        delegationKind: "alla fruizione",
         reason: rejectionReason ? ` Motivo: ${rejectionReason}.` : "",
       }))
       .exhaustive();
-    return `${delegateName} ha ${action} la delega in ${delegationKind}.${reason}`;
+    return `Ti informiamo che l'ente ${delegateName} ha ${action} la delega ${delegationKind} che il tuo ente gli ha conferito per l'e-service <strong>${eserviceName}</strong>. ${reason}`;
```

Comment:
it looks like the rejection reason isn't included in the text, so maybe we should remove it ("La delega è ora attiva." instead is still in the text correctly for the accepted case)